README.md

```diff
@@ -131,6 +132,29 @@ archive { '/tmp/test100k.db':
 }
 ```
 
+### Puppet URL
+
+Currently there isn't support for using ```puppet:\\``` style url in the source parameter.  It is still possible, but it would be done in two steps:
+
+```puppet
+$docs_filename = 'help.tar.gz'
+$docs_gz_path  = "/tmp/${docs_filename}"
+$homedir = '/home/myuser/'
+
+file {$docs_gz_path:
+  ensure => file,
+  source => "puppet:///modules/profile/${docs_filename}",
+}
+
+archive { $docs_gz_path:
+  path          => $docs_gz_path,
+  extract       => true,
+  extract_path  => $homedir,
+  creates       => "${homedir}/help" #directory inside tgz
+  require       => [ File[$homedir], File[$docs_gz_path] ],
```

Comment:
The $homedir is not necessary, it should be picked up by autorequire:

https://github.com/voxpupuli/puppet-archive/blob/master/lib/puppet/type/archive.rb#L164-L165
```
      self[:extract_path],
```